go/vt/vttablet/tabletmanager/vreplication/utils.go

```diff
@@ -33,8 +33,17 @@ import (
 
 const (
 	vreplicationLogTableName = "vreplication_log"
+	// This comes from the fact that the message column in the vreplication_log table is of type TEXT.
```

Comment:
Do we need this verbosity in the comments?  We can just say that this is to ensure message fits in the column, no?